.travis.yml

```diff
@@ -1,23 +1,66 @@
+sudo: false
+
 language: php
 
-php:
-  - 5.5
-  - 5.6
-  - 7
-  - hhvm
+cache:
+  directories:
+    - $HOME/.composer/cache
+    - vendor
+
+env:
+  global:
+    - COMPOSER_ARGS="--no-interaction --ignore-platform-reqs"
```

Comment:
This line is causing the tests to fail. AFAIK this ignores the PHP version as well. That's why it installs PackageVersions which has a PHP 7 dependency. And thus it fails on PHP 5.6 with the latest dependencies.